browser-switch/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     buildToolsVersion '28.0.2'
 
     defaultConfig {
-        minSdkVersion 15
+        minSdkVersion 21
```

Comment:
This change is actually part of PR #24 in branch `set-minSdkVersion-to-21`, but it appears in this PR as well, because I can't change the base branch of this PR to `set-minSdkVersion-to-21` which is only in my fork.